docs/api-testing/vault.md

```diff
@@ -69,40 +62,35 @@ Consider a scenario where an `/product` endpoint requires a specific `id` query
 
 While this is a perfectly valid request parameter, it can be hard to manage and update if you scale out your tests. Therefore rather than continuously hard-coding this value into the **Query Params** field, a safer and more efficient approach is to export this value into to a variable.
 
-Here is an example of how it could look like in the Project-specific **Vault**:
+Here is an example of how it could look like in the project-specific **Vault**:
```

Comment:
Change to "The following is an example of how it might look in the project-specific **Vault**"